static/js/jira-select-card-option.js

```diff
@@ -1,6 +1,48 @@
-$(".optionsCard").click(function (event) {
-	event.preventDefault();
-	$(".optionsCard").removeClass("selected");
-	$(event.currentTarget).addClass("selected");
-	$(".optionBtn").prop("disabled", false).attr("aria-disabled", "false").addClass("aui-button-primary");
+/* globals $, AP */
+const params = new URLSearchParams(window.location.search.substring(1));
+const jiraHost = params.get("xdm_e");
+
+function openChildWindow(url) {
+	const child = window.open(url);
+	const interval = setInterval(function () {
+		if (child.closed) {
+			clearInterval(interval);
+			AP.navigator.reload();
+		}
+	}, 100);
```

Comment:
Maybe change it to 1000 (1 sec) instead of 100ms? Am thinking preserving user computer resource (cpu/mem/battery) with trade-off of reloading the page a bit later.